src/SidebarBottom.tsx

```diff
@@ -0,0 +1,67 @@
+import { Badge as BaseBadge } from "@storybook/components";
+import { type API, useStorybookState } from "@storybook/manager-api";
+import { css, styled } from "@storybook/theming";
+import type { API_FilterFunction } from "@storybook/types";
+import pluralize from "pluralize";
+import React, { useState } from "react";
+
+import { IconButton } from "./components/IconButton";
+import { ADDON_ID } from "./constants";
+
+const Badge = styled(BaseBadge)(({ theme }) => ({
+  padding: "4px 8px",
+  fontSize: theme.typography.size.s1,
+}));
+
+const Button = styled(IconButton)(
+  {
+    "&:hover [data-badge], [data-badge=true]": {
+      background: "#E3F3FF",
+      borderColor: "rgba(2, 113, 182, 0.1)",
+      color: "#0271B6",
+    },
+  },
+  ({ active, theme }) =>
+    !active &&
+    css({
+      "&:hover": {
+        color: theme.color.defaultText,
+      },
+    })
+);
+
+const filterNone: API_FilterFunction = () => true;
+const filterWarn: API_FilterFunction = ({ status }) => status?.[ADDON_ID]?.status === "warn";
```

Comment:
What about errors?

Reply:
That doesn't seem to be a thing.

<img width="344" alt="Screenshot 2023-08-24 at 14 57 33" src="https://github.com/chromaui/addon-visual-tests/assets/321738/4124a472-5685-40a3-9b95-e14851b02d52">